lib/rspec/rails/fixture_file_upload_support.rb

```diff
@@ -23,6 +23,7 @@ class RailsFixtureFileWrapper
 
         class << self
           attr_accessor :fixture_path
+          attr_reader :file_fixture_path
```

Comment:
If this is only for the next version of rails, we need some kind of feature flag.

Reply:
Prior versions of Rails do not access `file_fixture_path` from within `ActionDispatch::TestProcess#fixture_file_upload`. Is there a benefit to not exposing the reader in those cases? Otherwise it feels a bit heavy handed.

Reply:
These is because the presence of this reader might be seen by other libraries feature detection and the wrong version of Rails assumed, we don't use it ourselves so theres no point having a defined reader that does nothing but might potentially false flag other libraries.

Reply:
Maybe I'm missing something, but it doesn't feel right to me to add code to support a hypothetical edge case such as a third party library leaning on this particular class in RSpec to determine a Rails version. However, this is your project, so please merge / modify / close out this PR as you wish.